app.py

```diff
@@ -4,9 +4,26 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=session.get_items())
+
+@app.route('/addit', methods=['GET', 'POST'])
```

Comment:
I'd split this into 2 separate methods (for GET & POST)

Reply:
removed GET as not required